### ISSUE TITLE
Assets for version alpha-1

### ISSUE THREAD

**Issue by itajaja** [itajaja/sco#8]
the assets for the first version of the game are in the following folder: https://github.com/itajaja/sco/tree/master/www/src/models/assets/alpha-1

there are still many assets missing but I am planning to complete it today.

@Kimerao0 can you please do the following three things:

- [ ] ensure that the values are correct
- [ ] translate the `name` properties to english in both buildings and units
- [ ] add a description to the `description` property of the units, and translate the `description` property in the buidlings

I know you probably don't understand much, but I am pretty sure that's intuitive

**Comment by itajaja** [comment]
### A small guide on how to update files using the github interface

once you are on a file, click on the small pencile on the top-right corner:

![image](https://cloud.githubusercontent.com/assets/697647/8541840/63f6a7e8-2491-11e5-8007-aeea46d12028.png)

you can edit, and preview the changes you made. Once you are done. on the bottom of the file make sure you click on "Create a new branch for this commit...":

![image](https://cloud.githubusercontent.com/assets/697647/8541855/7c29ee6a-2491-11e5-9fbb-e3e94c8c1a3b.png)

This is really important, so **PLEASE** remember to do that, and not commit on master. Let me know if you have any questions. cc'ing @theodred25, it will be useful also for you :)

**Comment by lucaBuratti** [comment]
The rusher is rushing in a rush. First, I hope you automatically generated these two json and not writed them by hand. Second the asset that you loaded relative to ships is alredy obsolete.

**Comment by itajaja** [comment]
I exported the json from the spreadsheet. It's a starting point. How obsolete? I took it from the spreadsheet? even so, it's fine, once the structure is in place at least I can get help from Ale or Marco to fill the gaps in the assets, while I focus on the logic. To me it's not rushing. The alternative would be to write the asset specification as an issue **and then** translating it to code. I guess we can do it directly in code to save time. I want to have a small working version by the end of the week

**Comment by lucaBuratti** [comment]
The combat phase seems to be drastically different from the game we are playing ( #3 )

**Comment by itajaja** [comment]
You are right, we should focus on the combat before, since it seems it's changed a lot
